### PR TITLE
Change to new driver download style

### LINKED ISSUE #73: Use new style of driver download from connection
Using two step build.

### PR DESCRIPTION
**Pull request by tomasfejfar**
Fixes #73

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,29 +17,6 @@ services:
       - SNOWFLAKE_DB_SCHEMA
       - SNOWFLAKE_DB_WAREHOUSE
   dev:
-    image: keboola/ex-db-snowflake
-    command: composer ci
+    <<: *base
```
Comment (pivnicek) [inline docker-compose.yml]: woah, this is some ancient Egyptian hack? :-)  I will use this going forward 👍

File: .travis.yml [hunk 2]
```diff
@@ -9,10 +9,7 @@ before_install:
   - docker-compose -v
 
 install:
-  - docker-compose run --rm drivers-download composer install
-  - docker-compose run --rm drivers-download php driver/downloadDriver.php
-  - docker-compose build app
-  - docker-compose run --rm app
+  - docker-compose run --rm app composer ci
```
Comment (pivnicek) [inline .travis.yml]: ok, I'm ok with moving it here from the docker-compose command property.  I agree that it is more clear.

File: docker-compose.yml (hunk 1 above)
Comment (tomasfejfar) [reply docker-compose.yml]: https://viewsourcecode.org/why/redhanded/bits/yamlSMergeKey.html

**Comment by tomasfejfar** [comment]
Should I merge/release?

**Comment by pivnicek** [comment]
Feel free to merge,  Those other PRs are merged also.  I will release when https://github.com/keboola/db-extractor-snowflake/pull/75 is merged (you can review it if you want :-) )

**Pull request merged**